htconfig.php

```diff
@@ -62,9 +62,9 @@
 
 // Server-to-server private message encryption (RINO) is allowed by default.
 // Encryption will only be provided if this setting is set to a non zero value
-// set to 0 to disable, 2 to enable, 1 is deprecated
+// set to 0 to disable, 3 to enable
```

Comment:
Could you describe the other options as well?

Reply:
I started a whole documentation page for RINO 😄